.jenkins/pytorch/win-test.sh

```diff
@@ -148,15 +148,15 @@ test_api.exe --gtest_filter="-IntegrationTest.MNIST*"
 EOL
 
 run_tests() {
-    if [ -z "${JOB_BASE_NAME}" ] || [[ "${JOB_BASE_NAME}" == *-test ]]; then
+    if [ -z "${BUILD_ENVIRONMENT}" ] || [[ "${BUILD_ENVIRONMENT}" == *-test ]]; then
         $TMP_DIR/ci_scripts/test_python_nn.bat && \
         $TMP_DIR/ci_scripts/test_python_all_except_nn.bat && \
         $TMP_DIR/ci_scripts/test_custom_script_ops.bat && \
         $TMP_DIR/ci_scripts/test_libtorch.bat
     else
-        if [[ "${JOB_BASE_NAME}" == *-test1 ]]; then
+        if [[ "${BUILD_ENVIRONMENT}" == *-test1 ]]; then
```

Comment:
Here as well

Reply:
I believe the Windows tests are no longer running at all after this change.

* Here's an example of `pytorch-win-ws2016-cuda9-cudnn7-py3-test1` that ran fully: https://ci.pytorch.org/jenkins/job/pytorch-builds/job/pytorch-win-ws2016-cuda9-cudnn7-py3-test1/22629/console
* Here's the latest `pytorch-win-ws2016-cuda9-cudnn7-py3-test1` run on master: https://ci.pytorch.org/jenkins/job/pytorch-builds/job/pytorch-win-ws2016-cuda9-cudnn7-py3-test1/22869//console

Note that the value of `${JOB_BASE_NAME}` used to be `pytorch-win-ws2016-cuda9-cudnn7-py3-test1` , but now `${BUILD_ENVIRONMENT}` is just `pytorch-win-ws2016-cuda9-cudnn7-py3` (without the `-test` suffix), so none of the conditions in the `run_tests()` function are matched.

Reply:
Filed Issue #17101